.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug Report
+about: Report errors and problems
+
+---
+
+<!--
+    The Symfony Code of Conduct applies to all the activity on this repository.
+    See https://symfony.com/doc/current/contributing/code_of_conduct/index.html
+-->
+
+**Symfony version(s) affected**
+x.y.z
+
+**Description**
+A clear and concise description of the problem.
```

Comment:
Why not commenting these placeholders so you just have to fill, not necessarily replace?